.github/CONTRIBUTING.txt

```diff
@@ -50,6 +43,10 @@ Here's the long and short of it:
 
       git push origin transform-speedups
 
+   * Enter your GitHub username and password (repeat contributors or advanced
+     users can remove this step by connecting to GitHub with SSH. See detailed
+     instructions below if desired).
```

Comment:
About three lines down, but out of this diff, there is a section which uses that phrase and links to the gitwash documentation.